doc/source/admin/conda_faq.rst

```diff
@@ -0,0 +1,332 @@
+|                          |                          |                          |
+|                          |                          | IMPORTANT : Due to a     |
+|                          |                          | current limitation in    |
+|                          |                          | conda, the total length  |
+|                          |                          | of the                   |
+|                          |                          |                          |
+|                          |                          | conda\_prefix and the    |
+|                          |                          | job\_working\_directory  |
+|                          |                          | path should be less      |
+|                          |                          | than 50 characters!      |
++--------------------------+--------------------------+--------------------------+
+| ``conda_auto_install``   | False                    | Set to True to instruct  |
+|                          |                          | Galaxy to look for and   |
+|                          |                          | install Conda packages   |
+|                          |                          | for missing tool         |
+|                          |                          | dependencies before      |
+|                          |                          | running a job            |
++--------------------------+--------------------------+--------------------------+
+| ``conda_auto_init``      | False                    | Set to True to instruct  |
+|                          |                          | Galaxy to install Conda  |
+|                          |                          | automatically if it      |
+|                          |                          | cannot find a local copy |
++--------------------------+--------------------------+--------------------------+
+
+*Table 1: Important configuration options for Conda in Galaxy*
+
+
+2. How do Conda dependencies work? Where do things get installed?
+
+In contrast to the old dependency system, which was used exclusively by
+Galaxy, Conda is a pre-existing, independent project. With Conda it
+is possible for an admin to install packages without touching Galaxy at
+all – managing your dependencies independently from Galaxy. Galaxy can
+handle these dependencies for you, but admins are not required to use
+Galaxy for dependency management.
+
+There are a few new config options in the galaxy.ini file (see Table
+1 or ``galaxy.ini.sample`` for more information), but by default Galaxy
+will install Conda (the package manager) and the required packages in
+the ``<tool_dependency_dir>/_conda/`` directory. In this directory,
+Galaxy will create an ``envs`` folder with all of the environments managed
+by Galaxy. Every environment contains a ``lib``, ``bin``, ``share``, and ``include``
+subdirectories, depending on the tool, and is sufficient to get a Galaxy
```

Comment:
I see - missed the article there. Thanks.

On Tue, Aug 23, 2016 at 4:48 PM Dannon Baker <notifications@github.com>
wrote:

> In doc/source/admin/conda_faq.rst
> <https://github.com/galaxyproject/galaxy/pull/2844#discussion_r75947588>:
>
> > +2. How do Conda dependencies work? Where do things get installed?
> > +
> > +In contrast to the old dependency system, which was used exclusively by
> > +Galaxy, Conda is a pre-existing, independent project. With Conda it
> > +is possible for an admin to install packages without touching Galaxy at
> > +all – managing your dependencies independently from Galaxy. Galaxy can
> > +handle these dependencies for you, but admins are not required to use
> > +Galaxy for dependency management.
> > +
> > +There are a few new config options in the galaxy.ini file (see Table
> > +1 or ``galaxy.ini.sample`` for more information), but by default Galaxy
> > +will install Conda (the package manager) and the required packages in
> > +the ``<tool_dependency_dir>/_conda/`` directory. In this directory,
> > +Galaxy will create an ``envs`` folder with all of the environments managed
> > +by Galaxy. Every environment contains a ``lib``, ``bin``, ``share``, and ``include``
> > +subdirectories, depending on the tool, and is sufficient to get a Galaxy
>
> You could go either way with 'Every environment contains a subdirectory'
> or 'Every environment contains subdirectories', but you can't use 'a' with
> 'subdirectories' here.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/galaxyproject/galaxy/pull/2844/files/01a24f6560be2d6cc9c6aef43e9665f140fc3640#r75947588>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABuxqpdUJfcd-6KmR4n8i60GjpwFAH6Wks5qi1yRgaJpZM4JrUBn>
> .
>